pkg/operator/e2e/context.go

```diff
@@ -97,7 +98,7 @@ func newTestContext(t *testing.T) *testContext {
 		t.Fatalf("create test namespace: %s", err)
 	}
 
-	op, err := operator.New(globalLogger, kubeconfig, nil, operator.Options{
+	op, err := operator.New(globalLogger, kubeconfig, prometheus.NewRegistry(), operator.Options{
```

Comment:
I'm fine with it in the e2e tests - but I don't see it being used anywhere in the operator currently. Did you add that somewhere?

Reply:
Used in #301